README.md

```diff
@@ -19,9 +19,10 @@ Our projects are community-built and welcome collaboration. 👍 Be sure to see
 <a href="http://slack.layer5.io"><img alt="Layer5 Service Mesh Community" src=".github/readme/images/community.png" style="margin-right:8px;padding-top:5px;" width="140px" align="left" /></a>
 
 <p>
-✔️ <em><strong>Join</strong></em> <a href="https://drive.google.com/open?id=1c07UO9dS7_tFD-ClCWHIrEzRnzUJoFQ10EzfJTpS7FY">weekly community meeting</a> on <a href="https://bit.ly/2SbrRhe">Fridays from 10am - 11am Central</a>.<br />
-✔️ <em><strong>Watch</strong></em> community <a href="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA?sub_confirmation=1">meeting recordings</a>.<br />
-✔️ <em><strong>Access</strong></em> the <a href="https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA">community drive</a>.<br />
+✔️ <em><strong>Join</strong></em> <a href="https://drive.google.com/open?id=1c07UO9dS7_tFD-ClCWHIrEzRnzUJoFQ10EzfJTpS7FY">Weekly Community Meeting</a> on <a href="https://bit.ly/2SbrRhe">Fridays from 10am - 11am Central</a>.<br />
+✔️ <em><strong>Watch</strong></em> Community <a href="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA?sub_confirmation=1">Meeting Recordings</a>.<br />
+✔️ <em><strong>Access</strong></em> the <a href="https://drive.google.com/drive/u/4/folders/0ABH8aabN4WAKUk9PVA">Community Drive</a>.<br />
+✔️ <em><strong>Discuss</strong></em> in the <a href="https://discuss.layer5.io">Community Forum</a>.<br />
```

Comment:
@devansh42 : Thanks for adding this! :slightly_smiling_face: 
Need some sort of changes!

Reply:
@amino19 I'm a little bit confused here, please help me out in resolving these pull request issues, like what exactly we want or if there is an example implementation.

Reply:
@devansh42 : I suggest you the changes above; for reference you can add points as similar to <a href="https://github.com/meshery/meshery/blob/master/README.md/"> this</a>!

Reply:
@devansh42 Could you ping me on slack?
We can get on a call tomorrow and I can help you resolve this :)

Reply:
@devansh42 : Thanks, the current commit looks good! :)

Reply:
@amino19 Shall we merge this request?

Reply:
@devansh42 : Clearly, @adithyaakrishna have a crown here!
Pl take this to you end :slightly_smiling_face: 

LGTM! :partying_face: